Makefile

```diff
@@ -0,0 +1,67 @@
+# by default, bmake will cd into ./obj first
+.OBJDIR: ./
+
+# Default install prefix
+PREFIX?=/usr/local
+
+# Use ninja, if it is installed
+_GENERATOR!=which ninja 2>/dev/null >/dev/null && echo Ninja || echo "'Unix Makefiles'"
```

Comment:
Note: We don't want to rely on `which`. Even POSIX [specifies](http://pubs.opengroup.org/onlinepubs/9699919799/) `command -v`. Use that instead because it's a shell builtin, not an external command that is mostly useless.

See 757a95123d8cae15fa1d9e2ccafac8b42bc315ab.

Reply:
Nice, I didn't realize `command` was portable.